src/core/display/Container.js

```diff
@@ -606,17 +606,23 @@ Container.prototype.renderCanvas = function (renderer)
 
 /**
  * Destroys the container
- * @param [destroyChildren=false] {boolean} if set to true, all the children will have their destroy method called as well
+ * @param [opts] {object} Options parameter
+ * @param [opts.children=false] {boolean} if set to true, all the children will have their destroy
+ *      method called as well. 'opts' will be passed on to those calls.
+ *
  */
-Container.prototype.destroy = function (destroyChildren)
+Container.prototype.destroy = function (opts)
 {
+    opts = opts || {};
```

Comment:
I've changed ```opts``` to ```options```. 

I'm can also allow booleans to be passed, the reason I didn't do it in the first place is because I didn't know how to do it for Text/Sprite. Shall I try to emulate the different old signature for each of them or always treat a boolean as "destroyChildren"? 

If we stick with the old signature, shall I throw a deprecation warning? I'm not sure it's a good idea to keep the old (and broken) signatures around forever.